README.md

```diff
@@ -110,6 +110,25 @@ To configure a custom check use the configuration below. This creates the corres
           - some_data: true
 ```
 
+##### Custom Python Checks
+
+To configure a custom check and to pass a Python check to the Playbook, please use the configuration below.  
+This requires the Datadog play / role to be part of a larger Playbook, with the value passed in being the relative filepath 
+to the actual task for [Linux](./tasks/agent-linux.yml) or [Windows](./tasks/agent-win.yml). 
+This is only available for Datadog v6+.  
```

Comment:
Would it be helpful to add a reference link for Play and Role (https://docs.ansible.com/ansible/latest/reference_appendices/playbooks_keywords.html#playbook-keywords)?

```suggestion
To pass a Python check to the playbook, use the configuration below. 

This configuration requires the Datadog play and role to be a part of the larger playbook where the value passed in is the relative file path to the actual task for [Linux](./tasks/agent-linux.yml) or [Windows](./tasks/agent-win.yml). 

This is only available for Agent v6+.  
```